packages/text-editor-utils/src/extensions/index.ts

```diff
@@ -22,18 +12,16 @@ lowlight.register("js", js);
 lowlight.register("ts", ts);
 
 export const extensions = [
-	Document,
-	Paragraph,
-	Bold,
-	ListItem,
-	BulletList,
-	Code,
+	StarterKit.configure({
+		blockquote: false,
+		codeBlock: false,
+		hardBreak: false,
+		heading: false,
+		horizontalRule: false,
+		dropcursor: false,
+		gapcursor: false,
+	}),
```

Comment:
Since importing tiptap extensions as external packages caused issues with resolving type information, we replaced them with the Starter Kit. We have selected the same extensions for use as before.

https://tiptap.dev/docs/editor/extensions/functionality/starterkit